crates/foundry/cheatcodes/src/config.rs

```diff
@@ -130,6 +165,7 @@ impl<HardforkT: HardforkTr> CheatsConfig<HardforkT> {
             labels,
             seed,
             allow_internal_expect_revert,
+            functions_internal_expect_revert,
         } = config;
 
         // TODO
```

Comment:
Question: I just happened to spot this. It's unrelated to your changes, but do we know what it's for?